### PR TITLE
Update device-onboarding-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated 5. with "A supported version of Microsoft 365 Apps is installed" as Perpetual office is not supported.

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10149

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

File: microsoft-365/compliance/device-onboarding-overview.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Make sure that the Windows devices that you need to onboard meet these requireme
    - [Hybrid Azure AD joined](/azure/active-directory/devices/concept-azure-ad-join-hybrid)
    - [AAD registered](/azure/active-directory/user-help/user-help-register-device-on-network)
 
-5. A supported version of Microsoft Office is installed and up to date. For the most robust protection and user experience, ensure Microsoft 365 Apps version 16.0.14701.0 or newer is installed.
+5. A supported version of Microsoft 365 Apps is installed and up to date. For the most robust protection and user experience, ensure Microsoft 365 Apps version 16.0.14701.0 or newer is installed.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/device-onboarding-overview.md]:
```suggestion
5. A supported version of Microsoft 365 Apps is installed and up to date. For the most robust protection and user experience, ensure Microsoft 365 Apps version 16.0.14701.0 or later is installed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**